java/core/src/main/java/com/microsoft/bond/protocol/Cloning.java

```diff
@@ -1,9 +1,24 @@
 package com.microsoft.bond.protocol;
 
 import java.io.ByteArrayInputStream;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
 import java.lang.reflect.Field;
+import java.nio.ByteBuffer;
+import java.nio.MappedByteBuffer;
+import java.nio.channels.FileChannel;
 
 class Cloning {
```

Comment:
>Cloning [](start = 6, length = 7)

Please declare all classes final, it is a useful hint to compiler optimizations.